.github/workflows/automatus-sle15.yaml

```diff
@@ -0,0 +1,189 @@
+name: Automatus SLE15
+on:
+  pull_request:
+    branches: [ master, 'stabilization*' ]
+env:
+  DATASTREAM: ssg-sle15-ds.xml
+jobs:
+  build-content:
+    name: Build Content
+    runs-on: ubuntu-latest
+    container:
+      image: registry.suse.com/bci/bci-base:latest
```

Comment:
In theory you would not need to use a SLE distro to build the content. It could be the same from other workflows (I guess it's fedora), since the built artifacts should be same disregarding the distro. But it can be. I've added a few suggestions and will add a test commit to trigger a rule to be tested.